scripts/st2bootstrap-deb.template.sh

```diff
@@ -114,14 +128,17 @@ install_st2_dependencies() {
   sudo apt-get update
 
   # Note: gnupg-curl is needed to be able to use https transport when fetching keys
-  sudo apt-get install -y gnupg-curl
+  if [[ "$SUBTYPE" != 'bionic' ]]; then
+    sudo apt-get install -y gnupg-curl
```

Comment:
This specific package doesn't present in Bionic?

Reply:
Yeah, it's not (https://packages.ubuntu.com/search?keywords=gnupg-curl) and https transport for curl adding keys to APT works by default.